webapi/src/main/twirl/queries/sparql/v1/generateInsertStatementsForCreateValue.scala.txt

```diff
@@ -60,39 +61,64 @@
             knora-base:valueHasString """@value.toString""" .
 
         @value match {
-            case textValue: TextValueV1 => {
 
-                @* Create a Standoff node for each StandoffPositionV1. *@
+            case textValueSimple: TextValueSimpleV1 => {
+                @* nothing to do *@
+            }
+
+            case textValueWithStandoff: TextValueWithStandoffV1 => {
+
+                @* Create a Standoff node for each StandoffTagV1. *@
 
-                @for(createStandoff <- textValue.prepareForSparqlInsert(newValueIri)) {
-                    ?newValue@valueIndex knora-base:valueHasStandoff <@createStandoff.standoffTagIri> .
+                @for((createStandoff, standoffNodeIndex) <- textValueWithStandoff.prepareForSparqlInsert(newValueIri).zipWithIndex) {
 
-                    <@createStandoff.standoffTagIri>
+                    @if(standoffNodeIndex == 0) {
```

Comment:
Wait, you are right! I made two different case statements! :-) `TextValueSimpleV1` and `TextValueWithStandoffV1`.